basyx.aasenvironment/basyx.aasenvironment-client/src/test/java/org/eclipse/digitaltwin/basyx/aasenvironment/client/TestFixture.java

```diff
@@ -100,7 +112,16 @@ public Submodel buildSmPre1() {
 	}
 
 	public SubmodelDescriptor buildSmPre1Descriptor() {
-		return DummySubmodelDescriptorFactory.createDummyDescriptor(SM_PRE1_ID, SM_PRE1_IDSHORT, smRepositoryBasePath, null);
+		return DummySubmodelDescriptorFactory.createDummyDescriptor(SM_PRE1_ID, SM_PRE1_IDSHORT, null, smRepositoryBasePath);
+	}
+
+	public SubmodelDescriptor buildSmPre1Descriptor_withMultipleInterfaces() {
+		LinkedList<org.eclipse.digitaltwin.basyx.submodelregistry.client.model.Endpoint> endpoints = new LinkedList<>();
+
+		endpoints.add(DummySubmodelDescriptorFactory.createEndpoint(SM_POS1_ID, smRepositoryBasePath, "OTHER-3.0"));
```

Comment:
I would change this to `SUBMODEL-REPOSITORY-3.0` and create an endpoint with just the path to the Submodel Repository because `OTHER-3.0` would not be spec conform even if this is just for testing purposes.